src/app/Instance/InstancePage/ProcessorsTabContent.tsx

```diff
@@ -142,6 +138,21 @@ export const ProcessorsTabContent = ({
     error: processorsError,
   } = useGetProcessorsApi();
 
+  const customToolbarElement = areProcessorsLoading ? (
+    <Skeleton width={"170px"} height={"35px"} />
+  ) : (
+    <Button
```

Comment:
Why do we check on the processors loading status? In theory the "create processor" button depends only on the bridge status, we could display it as soon as it's available. Like `!bridgeStatus ? ... `
Wdyt @ppadti @vpellegrino?

Reply:
You're right @kelvah. Button loading should depend on the bridge, and not on the list of processors (different API call).